testnet/upgrade.sh

```
@@ -5,12 +5,29 @@ cp start-apothem.sh start-apothem.sh.tmp || true
+    echo "  3. Compare env.example (upstream) with your .env_tmp"
+    echo "     and add any new variables to your .env file."
+    echo "  4. Resolve the conflict in env.example (or other files)"
+    echo "     then run 'git add <file>' and 'git merge --continue'."
+    echo "  5. Re-run this script once the conflict is resolved."
+    echo ""
+    echo "Your original .env has been preserved as .env_tmp"
+    mv start-apothem.sh.tmp start-apothem.sh || true
+    exit 1
+fi
 
 source .env_tmp
```

Shellcheck (0.11.0)

[warning] In POSIX sh, 'source' in place of '.' is undefined.

(SC3046)